client/app/components/authors/HeaderContainer.scss

```diff
@@ -60,24 +73,25 @@
         line-height: 0;
 
         @include Desktop() {
-            margin: $spacing-16 $spacing-18 0 $spacing-16;
+            margin: $spacing-20 $spacing-10 0 $spacing-8;
```

Comment:
I think to keep the same spacing as before (16px on the left of the pipe and 18px on the right), we need to make this margin $spacing-20 $spacing-6 0 $spacing-6. On the left we'll have 2px + 8px + 6px = 16px, and on the right it'll be 6px + 12 px = 18px

Reply:
With that margin, I get the feeling that the author name on hover is too close to the pipe `|`. You will find attached the differences between those two margins. We can discuss this later tho @antsgar , there's no rush
![Screen Shot 2021-04-05 at 09 36 57](https://user-images.githubusercontent.com/32345857/113574838-47da1c00-95f3-11eb-8f18-2d3a87bd9fee.png)
![Screen Shot 2021-04-05 at 09 37 32](https://user-images.githubusercontent.com/32345857/113574849-4b6da300-95f3-11eb-8364-ab40c1bd1e34.png)

Reply:
Fixed here https://github.com/standardnotes/listed/pull/150